syntax = "proto3";

message PersonMessage {
  int32 id = 1;
  string first_name = 2;
  string last_name = 3;
  string company_name = 4;
}

message Empty {}

message ListOfPersonMessages {
  repeated PersonMessage persons_list = 1;
}

service PersonService {
  rpc Get(Empty) returns (ListOfPersonMessages);
  rpc Create(PersonMessage) returns (PersonMessage);
}